LuckySpin/Views/Spinner/Index.cshtml

```diff
@@ -14,12 +15,14 @@
 </head>
 <body>
 
- <!-- //TODO: Use the DIJ TT.numberToText() along with a Model's luck value to display the lucky number as a word -->
-    <h1> Lucky Number </h1>
+    <!-- //TODO: Use the DIJ TT.numberToText() along with a Model's luck value to display the lucky number as a word -->
+    <h1> Lucky Number @TT.numberToText(Model.Luck)</h1>
     <button onclick='history.go(0)'>Spin</button>
- <!-- //TODO: Use Model values in the appropriate places in this template  -->
-    <div>2</div><div>7</div><div>9</div>
-    <img src="/Images/LuckySpinCoins.jpg" alt="Coins" style="display:block" />
+    <!-- //TODO: Use Model values in the appropriate places in this template  -->
+    <div>@Model.Numbers[0]</div>
+    <div>@Model.Numbers[1]</div>
+    <div>@Model.Numbers[2]</div>
```

Comment:
I like how you separated these by line, makes it easier to see the template at work